eng/pipelines/coreclr/perf-non-wasm-jobs.yml

```
@@ -111,6 +111,7 @@ jobs:
       platforms:
       - ios_arm64
       jobParameters:
+        buildArgs: -s clr.nativeaotruntime+clr.nativeaotlibs+libs -c $(_BuildConfig) -rc $(_BuildConfig)
```

nit: when all configurations are equal, `-c` is enough:
```suggestion
        buildArgs: -s clr.nativeaotruntime+clr.nativeaotlibs+libs -c $(_BuildConfig)
```